webapp/lib/controller.dart

```diff
@@ -128,25 +143,50 @@ void command(UIAction action, Data actionData) {
       ProjectConfigurationData projectConfigurationData = actionData;
       saveProjectConfiguration(projectConfigurationData.config);
       break;
+    case UIAction.saveProjectConfiguration:
+      savePackageConfiguration();
+      break;
+  }
+}
+
+enum NavAction {
+  ALLPROJECTS,
+  DASHBOARD
```

Comment:
prefer to use the camelcase for enums:

```suggestion
  allProjects,
  dashboard
```

Reply:
Noted and chanded. Also changed `TagOperation` to use camel case